python/GafferUI/ShufflePlugValueWidget.py

```diff
@@ -94,16 +95,19 @@ def hasLabel( self ) :
 	def childPlugValueWidget( self, childPlug ) :
 
 		for w in self.__row :
-			if w.getPlug().isSame( childPlug ) :
```

Comment:
Just checking that I understand this ... in the new version, we are now testing equality rather than isSame.  This is not a concern, because equality of Plugs actually uses pointer comparison anyway, so the original code would have actually been identical if it had said `w.getPlug() == childPlug` instead of using isSame?

Reply:
Yes, that's right. Really, we want to use `is`, but due to deficiencies in the `IECore.RefCounted` bindings, that doesn't work. Perhaps I should stick with `isSame()` so that we can easily switch everything back to `is` if we fix Cortex? I must admit, I'd got into the habit of using `==` for brevity, and because `isSame()` is a bit of a wart.

Reply:
I guess there isn't a convenient sugar like `in` for `is`?